contracts/contracts/RequestManager.sol

```diff
@@ -325,8 +327,8 @@ contract RequestManager is Ownable, LpWhitelist {
         );
         require(request.activeClaims == 0, "Active claims running");
 
-        request.withdrawInfo.filler = request.sender;
         request.withdrawn = true;
+        request.filler = request.sender;
```

Comment:
Why do we need to set the `filler` here? The filler should only be set by the L1 fill proof, right?

Reply:
you are right. this is a leftover from the time where we set the withdrawer which is now reflected by the `MAX_INT` assignment.

Reply:
This also needs to be removed in withdrawDeposit()